### PR TITLE
Doc/deal with backlog after rescheduling

### PR DESCRIPTION
**Pull request by L-M-Sherlock**
May close https://github.com/open-spaced-repetition/fsrs4anki-helper/issues/175

### CODE REVIEW

File: docs/tutorial.md [hunk 1]
```diff
@@ -90,6 +90,14 @@ This option controls whether the due dates of cards will be changed when you ena
 
 If rescheduling is enabled, the due dates of cards will be immediately changed. This often results in a large number of cards becoming due, so is not recommended when first switching from SM-2.
 
+If you insist on rescheduling cards and then see a huge backlog, here is the suggesion to deal with the backlog:
+
+1. Set review sort order to “relative overdueness”
+2. (Optional) set the deck option Maximum reviews/day
```
[the remark this answers is not in the document]
Comment (L-M-Sherlock) [reply docs/tutorial.md:96]: What about this suggestion: https://refold.la/roadmap/stage-1/c/srs-best-practices#Falling-Behind

[the remark this answers is not in the document]
Comment (Expertium) [reply docs/tutorial.md:96]: I feel like that's too clunky. @user1823 @brishtibheja suggestions are welcome

**Comment by Expertium** [comment]
New idea: only reschedule cards if at least one of these two conditions is met:
1) The new interval is different from the last one by more than 5%. Rationale: if we decrease the number of cards that are being rescheduled, it will also decrease the backlog for the user. If a card's interval has changed only slightly, it's fine to skip it.
Example: old interval = 100 days, new interval = 90 days. abs(100-90)/100=10/100/10%. Should be rescheduled.
2) The previous interval is >1 year long. Rationale: if a long time Anki user switches to FSRS, he may have cards with very long intervals that, ideally, should be due now, but aren't. This is bad. This means that optimal FSRS scheduling will kick too late, and these cards will spend >1 year not being scheduled by FSRS.

This should decrease the amount of due cards when "Reschedule cards on change" is used while simultaneously ensuring that cards that were under SM-2 and had long intervals won't have to wait for literal years to finally be scheduled by FSRS.
@L-M-Sherlock I think this should be implemented natively in Anki, as well as in the add-on rescheduling functions.
@user1823 your opinion is also welcome

**Comment by L-M-Sherlock** [comment]
> EDIT: I could've sworn I proposed something like 1) a long time ago. I wonder why it wasn't implemented.

Because it cannot reduce the backlog in our experiments:
- https://github.com/open-spaced-repetition/fsrs4anki-helper/issues/419 

I even keep the branch: https://github.com/open-spaced-repetition/fsrs4anki-helper/tree/Feat/reschedule-cards-only-if-the-new-interval-is-different-by-x

**Comment by user1823** [comment]
I think that this advice should be in [Anki's FAQs](https://faqs.ankiweb.net/) because backlogs are a common problem, irrespective of whether the user is using SM-2 or FSRS.

There is a lot of good advice in https://controlaltbackspace.org/catch-up/. It would be great if someone can summarize that advice (and add your own points) and then open a PR in https://github.com/ankitects/faqs/pulls.

By the way, [this](https://github.com/open-spaced-repetition/fsrs4anki-helper/issues/332#issuecomment-1868324367) is what I do (and advise to others) for dealing with backlogs.

**Comment by user1823** [comment]
As far as this PR is concerned, we can just add a link to the Anki FAQ page (once it is created).

**Comment by brishtibheja** [comment]
The docs mention this method: https://docs.ankiweb.net/filtered-decks.html?highlight=backlog#catching-up

A seperate FAQ page would be nice, but would take time before dae actually gets to it. Perhaps, link to some good external sources (refold, controlaltbackspace) for now? Advice on backlog seems to be abundant on the internet.

**Comment by user1823** [comment]
Users will be more willing to follow an "official" advice. So, a link to Anki FAQs would be better. Also, I don't think that there is any rush to add the link. So, we can wait until the new page is created.

**Comment by L-M-Sherlock** [comment]
What's the next step here?

**Comment by brishtibheja** [comment]
As we're right now trying to get the FAQ merged with the manual I don't think a seperate FAQ page is feasible (and we'd need to move it later).

I can write something up in a few days but it'll need to be in the manual. Question is, where in the manual?

**Comment by L-M-Sherlock** [comment]
I decide not to mention it in FSRS tutorial.

**Pull request closed without merging**